news/2024/2024-06-17-the-followpoint-dressurf-top-keyboard-smasher.md

```diff
@@ -0,0 +1,110 @@
+
+*What do you feel like most people don't truly realize about being a top player, especially in the highest ranks?*
+
+> There's undeniable pressure that I always expect and get, such as setting 1900pp's or trying to maintain current spot... So I always practice off-stream to get in shape, exploring pending tab on webpage, downloading new charts, rating them to play on stream later.
+
+*Who would you say are the best osu!mania player(s) at the moment that aren't you, and why?*
+
+> I think the term "best" is very subjective, if I see "best" in terms of inspiring and drawing players into the osu!mania scene, I must say [jhlee0133](https://osu.ppy.sh/users/140148) for 7K genre, [SillyFangirl](https://osu.ppy.sh/users/2288363) for 4K genre.
+
+*Is there any advice that you would give to people who are just starting out, or are wanting to improve to one day be like you?*
+
+> Well, it's going to be really hard to correlate your fingers to each column, but must be preceded first to get used to 7K playstyle. Once you become accustomed to the 7K playstyle and develop muscle memory for finger positioning, your progress will accelerate.
+
+*Do you think that peripherals matter when it comes to becoming better at the game, why or why not?*
+
+> Yes, if you use a keyboard that's high, I strongly recommend utilizing a wrist rest or at least a towel to alleviate potential wrist pain.
+>
+> A minimum requirement for keyboard I think is the ability to press 7+ keys properly at once. Additionally, 144hz monitor would be great advantage if you're going to step up into more extreme charts.
+
+*What drives you to keep improving despite being at the top?*
+
+> Until now, and still, I always set myself as rival and competed against myself, consistently challenging my previous scores to improve. I deliberately make charts' rate a little bit over my playlimit and play them when my hand's condition is very good. Of course, I get 91~92% on it, but I primarily concentrate on getting scores to battle on later.
+
+*What are some "endgame" scores or milestones you would want to achieve in the future?*
+
+> I want to develop at least 300+ bpm chordjack, 270~280+ bpm brackets or passing Stellium regular rice dan on 1.1x.
+
+*To finish things up, what does the future hold for you? Do you have any goals or anything else that you'd like to mention?*
+
+> First and foremost, I really want to thank all the supporters who have been by my side and those supports really drives me to continually improve more.
+>
+> It'll be really hard to play 7K genre at first, but trust me, they're really fun mod if you get used to playing in those columns! Never hesitate to seek advices from professional players, they'll gladly guide you and help you as you move on. Currently, my message box (DM) is closed, so you can ask me anything on Discord server by mentioning my name.
```

Comment:
```suggestion
> It'll be really hard to play 7K at first, but trust me, it's a really fun mode if you get used to playing in those columns! Never hesitate to seek advice from professional players, because they'll gladly guide you and help you as you move on. Currently, my message box (DM) is closed, so you can ask me anything on Discord server by mentioning my name.
```